src/Integration.Vsix.UnitTests/CFamily/PreCompiledHeaders/PchFilesDeleterTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
```

Comment:
Why didn't the "missing copyright" rule complain?

Reply:
Hm, that's very odd.
I see that we're still referencing the [correct QP](https://sonarcloud.io/organizations/sonarsource/quality_profiles/show?language=cs&name=C%23+QP+for+SonarLint), and the copyright rule is enabled... who should we talk to about this? SC folks? Or dotnet guild?

Reply:
... because it's a test file.

Reply:
🤦